pdp/static/js/__test__/calendars.test.js

```diff
@@ -0,0 +1,896 @@
+require('./globals-helpers').importGlobals([
+    { module: 'js/condExport', name: 'condExport' },
+    { module: 'js/classes.js', name: 'classes' },
+    { module: 'js/calendars.js', name: 'calendars' },
+], '../..');
+
+var formatDatetimeRaw = calendars.formatDatetimeRaw;
+var formatDatetimeISO8601 = calendars.formatDatetimeISO8601;
+var SimpleDatetime = calendars.SimpleDatetime;
+var Calendar = calendars.Calendar;
+var GregorianCalendar = calendars.GregorianCalendar;
+var Fixed365DayCalendar = calendars.Fixed365DayCalendar;
+var Fixed360DayCalendar = calendars.Fixed360DayCalendar;
+var CalendarFactory = calendars.CalendarFactory;
+var CalendarDatetime = calendars.CalendarDatetime;
+var CfTimeSystem = calendars.CfTimeSystem;
+var CfDatetime = calendars.CfDatetime;
+
+var epochYear = 1900;
```

Comment:
The PDP includes a few datasets that start at 1850. Will this be an issue?

Reply:
Actually, upon further investigation, the PDP only includes ONE timeset that starts then, which is probably a mistake, and I should investigate it. But, hypothetically, would we only be able to include post-1900 timesets in the future?

Reply:
This is a test. The default epoch year is 1800, for that very reason. But it is good to test with some other year.